README.md

```diff
@@ -209,7 +209,7 @@ make test
 
 ## 📊 Benchmarks
 
-Running a [Cairo program](./cairo_programs/benchmarks/fibonacci_1000_multirun.cairo) that gets the 1000th Fibonacci number we got the following benchmarks:
+Running a [Cairo program](./cairo_programs/benchmarks/big_fibonacci.cairo) that gets the 1.5 millionth Fibonacci number we got the following benchmarks:
```

Comment:
isnt 2 millionth?

Reply:
Nope, it's 1.500.000:
```cairo
    fib(1, 1, 1500000);
```

Reply:
ahh sorry, got confused by the factorial jeje